build/ferretdb/evaluation/supervisord.conf

```diff
@@ -0,0 +1,18 @@
+[supervisord]
+nodaemon=true
+logfile=/dev/null
+logfile_maxbytes=0
+
+[program:postgres]
+command=/usr/local/bin/docker-entrypoint.sh postgres
+; fast shutdown https://www.postgresql.org/docs/current/server-shutdown.html
+stopsignal=INT
+stdout_logfile=/dev/stdout
```

Comment:
We probably should log to stderr, not stdout

Reply:
Also updated for logs to go to stderr